acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -475,7 +475,7 @@ def stop_sleep_process(targets, seconds_to_sleep, accept_no_pid_found = false)
     pids.each do |pid|
       target['platform'] =~ /win/ ?
         on(target, "taskkill /F /pid #{pid}") :
-        on(target, "kill -s TERM #{pid}")
+        on(target, "kill -15 #{pid}")
```

Comment:
Can you add a comment explaining `-15` is `TERM`?

Reply:
Done.